lua/telescope/builtin/init.lua

```diff
@@ -304,17 +304,31 @@ builtin.lsp_workspace_symbols = require('telescope.builtin.lsp').workspace_symbo
 builtin.lsp_dynamic_workspace_symbols = require('telescope.builtin.lsp').dynamic_workspace_symbols
 
 --- Lists LSP diagnostics for the current buffer
+--- - Fields:
+---   - `All severity flags can be passed as `string` or `number` as per `:vim.lsp.protocol.DiagnosticSeverity:`
 --- - Default keymaps:
 ---   - `<C-l>`: show autocompletion menu to prefilter your query with the diagnostic you want to see (i.e. `:warning:`)
 ---@param opts table: options to pass to the picker
 ---@field hide_filename boolean: if true, hides the name of the file in the current picker (default is false)
+---@field severity string: filter diagnostics by severity name (string) or id (number)
+---@field severity_limit string: filter for diagnostics equal or more severe wrt severity name (string) or id (number)
```

Comment:
What does `wrt` stand for here?

Reply:
`with respect to` - I've abbreviated it to keep the line short enough so linter wouldn't complain :sweat_smile: 

Happy to write it in full if that's preferred.